.gitignore

```diff
@@ -4,9 +4,6 @@
 **/debug/
 **/target/
 
-# Libs don't ship cargo.lock
-Cargo.lock
```

Comment:
The workspace shouldn't ship a Cargo.lock file

What could be done is the following: 
* Exclude the cli from the workspace
* Version-control the Cargo.lock produced at `cli/Cargo.lock` - Crates that are not part of a workspace get their own Cargo.lock